Tests/MongoSwiftTests/DNSSeedlistTests.swift

```diff
@@ -61,21 +57,35 @@ final class DNSSeedlistTests: MongoSwiftTestCase {
             asType: DNSSeedlistTestCase.self
         )
         for (fileName, testCase) in tests {
-            let topologyWatcher = TopologyDescriptionWatcher()
-
             // TODO: DRIVERS-796 or DRIVERS-990: unskip this test
             guard fileName != "txt-record-with-overridden-uri-option.json" else {
-                return
+                continue
+            }
+
+            // this particular test case requires SSL is disabled. see DRIVERS-1324.
+            let requiresTLS = fileName != "txt-record-with-overridden-ssl-option.json"
+
+            // TLS requirement for this test case is not met.
+            guard (requiresTLS && MongoSwiftTestCase.ssl) || (!requiresTLS && !MongoSwiftTestCase.ssl) else {
+                continue
```

Comment:
for this skip and above, I think we should add a print statement mentioning that its being skipped to ensure the cases are being skipped under the appropriate conditions and that we have a constant reminder that we should unskip these at some point.

Reply:
done